caikit/runtime/servicers/info_servicer.py

```diff
@@ -38,12 +44,65 @@ class InfoServicer:
     """This class contains the implementation for retrieving information about the
     library and services."""
 
+    def GetModelsInfo(
+        self, request: ModelInfoRequest, context  # pylint: disable=unused-argument
+    ) -> ModelInfoResponse:
+        """Get information on the loaded models for the GRPC server
+
+        Args:
+            request: ModelInfoRequest
+            context
+
+        Returns:
+            models_info: ModelInfoResponse
+                DataObject containing the model info
+        """
+        return self.get_models_info_impl().to_proto()
```

Comment:
NIT: We've tended to use `_get_models_info` rather than `get_models_info_impl` naming to indicate private impl functions

Reply:
Yeah, that's my preferred as well.  I saw that the info servicer was already using `get_runtime_info_impl` so I tried to follow the same convention. I'll update both to use underscores